### ISSUE TITLE
When users hover a dragged node over a closed parent node(lazy children)

### ISSUE THREAD

**Issue by jaabiri** [fritz-c/react-sortable-tree#229]
Hello @fritz-c 
i have implemented the lasy loading on a parent node to request their children data and then get expanded on clicking on it,
what i really want to do is to make a node requesting it's children and expand them once they become ready while i'm dragging one other node and hovering with it over the first one, so i could drop it in the depth level i would, and this is an explanation that i found  clear enought:
![34426786-01a9f194-ebef-11e7-9d2d-2ad48d02d795](https://user-images.githubusercontent.com/7157261/34749601-a1a65036-f5a2-11e7-9d24-6727e6d5eec6.gif)

**Comment by s-dezign** [comment]
Agreed. Need to implement the same thing. If anyone can share the solution — it will be awesome! Thanks.

**Comment by 8-bitTom** [comment]
👍  We also need this featur

**Comment by wuweiweiwu** [comment]
I can look at it! On closed branches. set a hover time that will expand the branch or something similar
